code-challenges.test.js

```diff
@@ -50,3 +125,75 @@ const hand4 = [7, 2, 7, 2, 7]
 // Expected output: true
 
 // b) Create the function that makes the test pass.
+// FXN PSEUDO
+// initialize isFullHouse to take one input array
+// initialize a map
+// iterate over hand
+// if current number is not in map
+    // add it 
+    // if count incremented > 2
+    // return false
+// if it is in map
+    // increment the map value
+
+// if map.values array includes a 3 or a 2
+    // return true
+// else 
+    // return false
+
+const isFullHouse = (hand) => {
+    if (hand.length < 5) return false
+
+    let counts = {}
+    let quit = 0
+
+    // for (let i = 0; i < hand.length; i++) {
+    //     if (counts[hand[i]]) {
+    //         ++counts[hand[i]]
+    //     } else {
+    //         if (++quit > 2) return false
+    //         counts[hand[i]] = 1
+    //     }
+    // }
+    
+    // for...of loop answer
+    for (const card of hand) {
+        if (counts[card]) {
+            ++counts[card]
+        } else {
+            if (++quit > 2) return false
+            counts[card] = 1
+        }
+    }
+
+    if (Object.values(counts).includes(2)) return true
+
+    return false
+}
+// Attempt to do challenge 3 using Map()
```

Comment:
This is the best thing that I got in the habit of doing: documenting the process and trials. These assessments are for you to have access to always, so you can always look back at the different approaches you have tried before. Great notes for yourself in the future.